docs/modules/plugins/partials/locator-header.adoc

```diff
@@ -13,3 +13,42 @@ By. prefix is optional.
 . `visibility` - *[optional]* visibility of element (visible by default)
 . `filterType` - *[optional]* type of the filter
 . `filterValue` - *[required if filter type defined]* value of the filter
+
+
+=== Reusable locators
+
+Feature allows to define custom reusable locator types via properties. These new locator types should rely on one of the already existing ones.
+
+.Properties
+----
+ui.locator.<locatorName>.pattern
+ui.locator.<locatorName>.locator-type
+----
+
+. `locatorName` - The name of the reusable locator.
+. `pattern` - The pattern of the reusable locator. Parameter placeholder is `%s`. If you need to use `%` as a part of the pattern use `%%`.
```

Comment:
it limits users just to %s, should not we add link to string formatting in java for "advanced" learning because things like positional parameter might me useful e.g.
ui.locator.tut.locator-type=xpath
ui.locator.tut.pattern=//*[@label='%1$s' or @name='%1$s']
By.tut(Text)